code-review/README.md

```diff
@@ -0,0 +1,119 @@
+
+Code Review Mindset
+-------------------
+
+It's important to separate our self-worth from the code we write so that we are always open to feedback that will help us improve.
+
+When reviewing code that changes user facing components it is important that reviews play the role of a User Advocate and consider how the change will impact our users, for example:
+* Is the copy written in such a way that it would be understand by someone who has learned English as their second language?
+* Is the change consistent with the user's mental model of our system?
+
+Everyone
+--------
+
+* Accept that many programming decisions are opinions. Discuss tradeoffs
+  which you prefer, and reach a resolution quickly.
+* Ask questions; don't make demands.
+    * *What do you think about naming this `:user_id`?*
+* Ask for clarification. 
+    * *I didn't understand. Can you clarify?*
+* Avoid selective ownership of code. 
+    * Avoid these words: "mine", "not mine", "yours"
+* Avoid using terms that could be seen as referring to personal traits.
+  ("dumb", "stupid"). Assume everyone is attractive, intelligent, and
+  well-meaning.
+* Be explicit. Remember people don't always understand your intentions online.
+* Be humble. 
+    * *I'm not sure - let's look it up.*
+* Don't use hyperbole. 
+    * Avoid "always", "never", "endlessly", "nothing"
+* Don't use sarcasm.
+* Keep it real. If emoji, animated gifs, or humor aren't you, don't force
+  them. If they are, use them with aplomb.
+* Talk in person if there are too many "I didn't understand" or "Alternative
+  solution:" comments. Post a follow-up comment summarizing offline discussion.
+
+Having Your Code Reviewed
+-------------------------
+
+* Be grateful for the reviewer's suggestions.
+  * *Good call. I'll make that change.*
+* Don't take it personally. The review is of the code, not you.
+* Explain why the code exists. 
+  * *It's like that because of these reasons. Would it be more clear if I rename this class/file/method/variable?*
+* Extract some changes and refactorings into future tickets/stories.
+* Link to the code review from the ticket/story.
+  * *Ready for review: https://github.com/organization/project/pull/1*
+* Push commits based on earlier rounds of feedback as isolated commits to the
+  branch. Do not squash until the branch is ready to merge. Reviewers should be
+  able to read individual updates based on their earlier feedback.
+* Seek to understand the reviewer's perspective.
+* Try to respond to every comment.
+* Wait to merge the branch until Continuous Integration (CircleCI)
+  tells you the test suite is green in the branch.
+* Merge once you feel confident in the code and its impact on the project.
+
+Reviewing Code
+--------------
+
+Understand why the code is necessary (bug, user experience, refactoring). Then:
+
+* Communicate which ideas you feel strongly about and those you don't.
```

Comment:
Is this in line with my comment about non-blocking and blocking? Is it better to be explicit, rather than "I feel strongly"?